includes/manage_collection.inc

```diff
@@ -394,6 +394,9 @@ function islandora_basic_collection_share_children_form(array $form, array &$for
  *   The Drupal form state.
  */
 function islandora_basic_collection_share_children_form_validate(array $form, array &$form_state) {
+  if (empty(array_filter($form_state['values']['children']))) {
```

Comment:
Travis (in PHP 5.4 and below) doesn't like using "function return value in write context" so once that's fixed, this PR looks good.

Reply:
@jordandukart sadly it's the `'empty(named_function(..` . same with`isset()`, both are language constructs and act like that in old php. passing array_filter($form_state['values']['children'])) to a temporary var and the doing empty() check over it should solve the issue